test/tests/models/tx/Test_PegOutCoin.cpp

```diff
@@ -1 +1,46 @@
-// TODO: Test Serialization/Deserialization (Raw & JSON)
\ No newline at end of file
+#include <catch.hpp>
+
+#include <mw/crypto/Crypto.h>
+#include <mw/models/tx/PegOutCoin.h>
+
+TEST_CASE("Tx Peg-Out Coin")
+{
+    uint64_t amount = 123;
+    Bech32Address address = Bech32Address::FromString("bc1qc7slrfxkknqcq2jevvvkdgvrt8080852dfjewde450xdlk4ugp7szw5tk9");
```

Comment:
bc1qc7slrfxkknqcq2jevvvkdgvrt8080852dfjewde450xdlk4ugp7szw5tk9 is not valid litecoin bech32 address instead it looks like a bitcoin bech32 address.

Reply:
Yes, it's perfectly fine to use this address. Bech32 addresses have something called an "HRP" (Human readable part) that we set to different values for each coin *by convention* only. So the hrp of this string won't be the same one used by ltc wallets, but that's ignored by the protocol anyway. The important part is the actual pubkey that's encoded in it.

Reply:
ok sounds good.
Even for consistency? it's for litecoin so shouldn't we use litecoin address?

Reply:
Sure, this one is just a bech32 address that's used as an example bech32 address in multiple places like this: https://en.bitcoin.it/wiki/Bech32

Do you happen to know a ltc bech32 address we can change it to? If not, I can generate one.

Reply:
This one tltc1qh50sy0823vxn4l9zk2820w4fuj0q4fgv48ma6c ?